ccd-definition/FixedLists/GeneralApplicationTypesGAspec.json

```diff
@@ -34,41 +34,5 @@
     "ListElementCode": "RELIEF_FROM_SANCTIONS",
     "ListElement": "Relief from sanctions",
     "DisplayOrder": 6
-  },
```

Comment:
In my opinion one file should contain all values which will be used in nonprod environment. Including new changes. One file should be wich contains only prod configuration. For us ATM all files containing name "...GAspec" are like nonProd files